File: webapp-poc/src/main/java/com/fingerprint/urltagging/service/URLTaggingService.java
package com.fingerprint.urltagging.service;

import java.util.List;

import com.fingerprint.object.ResponsePaginationForm;
import com.fingerprint.urltagging.object.URLForm;

public interface URLTaggingService {
	
	public ResponsePaginationForm<URLForm> getAllURLs(String url, String page, String limit) throws Exception;
	public ResponsePaginationForm<URLForm> save(List<URLForm> urlForms) throws Exception;
}

File: webapp-poc/src/main/webapp/resources/scripts/services/urlmanagement/URLTaggingService.js
'use strict';

angular.module('fingerPrintApp')
    .factory('urlTaggingService', function($http) {
  
    	function handleSuccess(data) {
            return data;
        }

    	function handleError(error) {
            return function () {
                return { success: false, message: error };
            };
        }
   return {
	   getAllURLs: function(url, page, limit) {
           return $http.get('urltagging/getAll?url='+url+'&page='+page+"&limit="+limit).then(handleSuccess, handleError('Error getting all users'));
       },
       save: function(items) {
           return $http.post('urltagging/save', items).then(handleSuccess, handleError('Error creating user'));
       }
       
   }
});
